cryptoprotocol/src/main/java/org/cryptimeleon/incentivesystem/cryptoprotocol/Util.java

```diff
@@ -25,11 +25,11 @@
     public static Zn.ZnElement hashGamma(Zn zn, BigInteger k, GroupElement dsid, Zn.ZnElement tid, GroupElement cPre0, GroupElement cPre1) {
         var hashfunction = new HashIntoZn(zn);
         var accumulator = new ByteArrayAccumulator();
-        accumulator.append(k.toByteArray());
-        accumulator.append(dsid);
-        accumulator.append(tid);
-        accumulator.append(cPre0);
-        accumulator.append(cPre1);
+        accumulator.escapeAndAppend(k.toByteArray());
```

Comment:
`escapeAndAppend`

Reply:
Sorry. I meant to say `escapeAndSeparate`. 
Without separator symbol, escaping doesn't really help much.